I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 144, in script
    jump talk_voice
  File "game/script.rpy", line 144, in script
    jump talk_voice
Exception: Possible infinite loop.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\workSpace\renpy-8.0.3-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "D:\workSpace\renpy-8.0.3-sdk\renpy\main.py", line 660, in main
    run(restart)
  File "D:\workSpace\renpy-8.0.3-sdk\renpy\main.py", line 146, in run
    renpy.execution.run_context(True)
  File "D:\workSpace\renpy-8.0.3-sdk\renpy\execution.py", line 938, in run_context
    context.run()
  File "game/script.rpy", line 144, in script
    jump talk_voice
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "game/script.rpy", line 144, in script
    jump talk_voice
  File "D:\workSpace\renpy-8.0.3-sdk\renpy\execution.py", line 61, in check_infinite_loop
    raise Exception("Possible infinite loop.")
Exception: Possible infinite loop.

Windows-10-10.0.22621 AMD64
Ren'Py 8.0.3.22090809
Amadeus System version2.0
Thu Feb  9 12:15:58 2023
